src/lua/lua_text.c

```diff
@@ -194,7 +194,7 @@ LUA_FUNCTION_DEF (text, base32);
  * @method rspamd_text:base64([line_length, [nline, [fold]]])
  * Returns a text encoded in base64 (new rspamd_text is allocated)
  *
- * @param {number} line_length return text splited with newlines up to this attribute
+ * @param {number} line_length return text split with newlines up to this attribute
```

Comment:
This is perhaps a notable change (the alternative is `splitted`)

Reply:
It doesn't change API from what I see, so no objections for whatever option.